e2e/app/page/notebook-page.ts

```diff
@@ -51,7 +51,7 @@ export default class NotebookPage extends AuthenticatedPage {
       console.warn(`Reloading "${this.documentTitle}" because cannot find the Run button`);
       await this.page.reload({waitUntil: ['networkidle0', 'load']});
     }
-    await this.waitForKernelIdle(300000); // 5 minutes
```

Comment:
nitpick: if you do this as `10 * 60 * 1000` instead of `60000` it makes it more obvious that it's 10 minutes

Reply:
👍